apps/shipments/management/commands/replace_shipments.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import sys
+
+from django.core.management.base import BaseCommand
+
+from django.conf import settings
+from shipchain_common.exceptions import RPCError
+
+from apps.eth.models import EthAction
+from apps.eth.rpc import EventRPCClient
+from apps.jobs.models import AsyncJob
+from apps.jobs.tasks import AsyncTask
+from apps.shipments.models import Shipment
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class Command(BaseCommand):
+    help = 'Moves shipment(s) to etherscan'
+    rpc_client = None
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--shipment_id',
+            type=str,
+            default=None,
+            help='Delete poll instead of closing it',
+        )
+
+    def _convert_shipment(self, shipment):
+        if not self.rpc_client:
+            self.rpc_client = EventRPCClient()
+            try:
+                self.rpc_client.unsubscribe("LOAD", settings.LOAD_VERSION)
+            except RPCError as exc:
+                if 'Error: EventSubscription not found' in exc.detail:
+                    return
```

Comment:
If there was no preexisting `EventSubscription` found, we don't process any shipments?  Why would we not continue processing shipments in this scenario?  The net outcome is the same if we successfully unsubscribe as it would be if there was no subscription in the first place.